src/StoreApi/Utilities/ValidationUtils.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Automattic\WooCommerce\StoreApi\Utilities;
+
+use \Exception;
+use Automattic\WooCommerce\StoreApi\Exceptions\RouteException;
+
+/**
+ * ValidationUtils class.
+ * Helper class which validates and update customer info.
+ */
+class ValidationUtils {
+	/**
+	 * Get list of states for a country.
+	 *
+	 * @param string $country Country code.
+	 * @return array Array of state names indexed by state keys.
+	 */
+	public function get_states_for_country( $country ) {
+		return $country ? array_filter( (array) \wc()->countries->get_states( $country ) ) : [];
```

Comment:
If we're keeping this method just for the context of this class, you might be able to move the `wc_strtoupper` part here too to avoid repetition below.

Reply:
Done